grammar.js

```diff
@@ -66,6 +62,7 @@ module.exports = grammar(C, {
     [$._binary_fold_operator, $._fold_operator],
     [$.expression_statement, $.for_statement],
     [$.init_statement, $.for_statement],
+    [$._function_declarator_seq],
```

Comment:
What was the new conflict here?

Reply:
` repeat($.attribute_specifier)` after noexcepts